book/src/1.welcome/2.package-decomposition.md

```diff
@@ -0,0 +1,53 @@
+# Package Decomposition
+The SDK is broken down into the following packages.
```

Comment:
Keep in mind this will change when C# comes in (very soon). Any ideas on how this might evolve?

Reply:
are the learn docs our source for language-specific questions? if thats the case, maybe we can have this included there instead

Reply:
I believe we will be adding the Tabs plugin to mdbook which would allow us to put different lists of the packages per language like

TypeScript | C# | Python

Reply:
hmm in that case, should we delete this for now until we introduce tabs?